css/style.css

```diff
@@ -36,44 +29,212 @@ Developer @GA in NYC
   padding: 0 0 10px 10px; /* note the padding around a right floated image */
 }
 
-.alignleft {
-  float: left;
-  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+  .alignleft {
+    float: left;
+    padding: 0 10px 10px 0; /* note the padding around a left floated image */
+  }
+
+  /******************************************
+  /* BASE STYLES
+  /*******************************************/
+
+  body {
+       color: #000;
+       font-size: 24px;
+       font-weight: 600;
+       line-height: 1.4;
+       font-family: 'Open Sans', sans-serif;
+       background-color: #f0efef;
+       color: #033048;
+       border:solid #033048 1px;
+       margin: auto;
+       width: 1024px;
+  }
+
+
+  header{
+      background-color:#033048;
+      color:#f9e42e;
+      width: 100%;
+      margin: auto;
 }
 
-/******************************************
-/* BASE STYLES
-/*******************************************/
-
-body {
-     color: #000;
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
+nav{
+  padding-left: 20px;
+  display: inline;
```

Comment:
since you're floating this element anyway, there's no need to set this to `display: inline`